File: src/question/month08/day0825/ms_0825.cpp
#include <iostream>

using namespace std;

int main() {
	int students[100], i = 0;
	int nss[10] = { 0, }; // Number of Students by Score

	while (i < 100) {
		int temp;

		scanf_s("%d", &temp);

		if (temp != 0)
			nss[(temp / 10) - 1]++;
		else break;

		i++;
	}

	for (i = sizeof(nss) / sizeof(nss[0]) - 1; i >= 0; i--)
		if (nss[i] != 0)
			cout << (i + 1) * 10 << " : " << nss[i] << " person" << endl;

	return 0;
}

File: src/question/month08/day0825/ms_0825.cs
using System;

namespace algorithm_42
{
    class ms_0825
    {
        static void Main(string[] args)
        {
            int[] students = new int[100];
            int[] nss = new int[10]; // number of students by score
            int i = 0;

            while(i < studnets.Length) 
            {
                int temp = Convert.ToInt32(Console.ReadLine());

                if(temp != 0) 
                    nss[(temp/10)-1]++;
                else break;
                
                i++;
            }

            for(i = nss.Length-1; i >= 0; i--) 
                if(nss[i] != 0)
                    Console.WriteLine($"{(i+1)*10} : {nss[i]} person");
        }
    }
}
